server/datastore/mysql/migrations/tables/20241021224359_AddExecutionStatusToHostSoftwareInstalls.go

```diff
@@ -0,0 +1,54 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241021224359, Down_20241021224359)
+}
+
+func Up_20241021224359(tx *sql.Tx) error {
+	if _, err := tx.Exec(`
+ALTER TABLE host_software_installs
```

Comment:
Let's add a comment for future readers about why we have another generated column that looks 95% the same as the other one.

Reply:
Done in [6ad503f](https://github.com/fleetdm/fleet/pull/23072/commits/6ad503f62b456a4796cd8f9d898d78355e33c042)